src/components/teaser/teaser.scss

```diff
@@ -91,9 +93,11 @@
   }
 }
 
-.sbb-teaser__lead {
-  @include sbb.clamp-lines($lines: 2);
+.sbb-teaser__chip {
+  margin-block-end: var(--sbb-spacing-fixed-1x);
+}
 
+.sbb-teaser__lead {
   // Overwrite sbb-title default margin
   margin: 0;
 }
```

Comment:
there is one more clamp lines which should be removed on sbb-teaser__description